src/System.Collections.Specialized/src/System/Collections/Specialized/ListDictionary.cs

```diff
@@ -493,6 +492,7 @@ public void Reset()
         }
 
         [Serializable]
+        [System.Runtime.CompilerServices.TypeForwardedFrom("System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
```

Comment:
@weshaggard @jkotas Without this TypeForwardedFrom attribute for this nested type I would get a blob which contains the assembly information of System.Collection.Spezialized, Version=... when serializing it in Core

```
�����������
���ISystem, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089
���aSystem.Collections.Specialized, Version=4.1.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a���-System.Collections.Specialized.ListDictionary���headversioncountcomparer��<System.Collections.Specialized.ListDictionary+DictionaryNode���System.CultureAwareComparer���	���������	������<System.Collections.Specialized.ListDictionary+DictionaryNode���keyvaluenext<System.Collections.Specialized.ListDictionary+DictionaryNode���������keyx���valuex
���System.CultureAwareComparer���
_compareInfo
_ignoreCase� System.Globalization.CompareInfo	������� System.Globalization.CompareInfo���m_name
m_sortVersion System.Globalization.SortVersion	����
```

With my change:
```
�����������
���ISystem, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089���-System.Collections.Specialized.ListDictionary���headversioncountcomparer��<System.Collections.Specialized.ListDictionary+DictionaryNode���System.CultureAwareComparer���	���������	������<System.Collections.Specialized.ListDictionary+DictionaryNode���keyvaluenext<System.Collections.Specialized.ListDictionary+DictionaryNode���������keyx���valuex
���System.CultureAwareComparer���
_compareInfo
_ignoreCase� System.Globalization.CompareInfo	������� System.Globalization.CompareInfo���m_name
m_sortVersion System.Globalization.SortVersion����
```

As you told me that we don't need type forwards on nested types am curious why it isn't working here without it?